weave/weave_init.py

```diff
@@ -95,7 +95,17 @@ def init_weave(project_name: str) -> InitializedClient:
     autopatch.autopatch()
 
     username = get_username()
-    init_message.print_init_message(username, entity_name, project_name)
+    try:
+        min_required_version = (
+            remote_server.server_info().min_required_weave_python_version
+        )
+    except Exception:
```

Comment:
Hmm, catch-all kinda sucks... this will catch SyntaxError. But I get you're worried that there are lots of different kinds of exceptions this could encounter

Reply:
yeah exactly - i really don't want to crash use code here. Added a TODO to tighten it